### ISSUE TITLE
fatal error: linuxutils.h: No such file or directory

### ISSUE THREAD

**Issue by xiota** [quotient-im/Quaternion#915]
In commit 63bc6fedfb81eda53a4f9ea527e11b0a1ff3cafe, `linuxutils.h` is renamed to `desktop_integration.h`, but an instance was forgotten, resulting in error:
```
client/mainwindow.cpp:14:10: fatal error: linuxutils.h: No such file or directory
```

https://github.com/quotient-im/Quaternion/blob/c3eb1bc26084cd4635aad8e311ff086ff00cd465/client/mainwindow.cpp#L14

**Comment by KitsuneRal** [comment]
Gah, I thought I had pushed it :( thanks for reporting

**Comment by KitsuneRal** [comment]
Fixed by 2d2c94b0e5506206ef7fa20c51d0a10726716ad7
